triage/nw_prj_artifact_compare.py

```
@@ -0,0 +1,143 @@
+
+    rpt.admin_authority = "manual_admin_scratch"
+    if inputs.official_admin_path and not inputs.admin_scratch_path:
+        rpt.admin_authority = "official_admin_workbook"
+        rpt.warnings.append("official_admin_without_scratch")
+
+    prior: Dict[str, DashboardRow] = {}
+    if inputs.dashboard_path:
+        prior = {row_key(r): r for r in load_dashboard_rows(inputs.dashboard_path)}
+
+    # Seed from prior dashboard with carry-forward
+    for key, row in prior.items():
```

Ruff (0.15.14)

[warning] Loop control variable `key` not used within loop body

(B007)